src/components/application_manager/src/commands/mobile/perform_interaction_request.cc

```diff
@@ -420,17 +424,6 @@ bool PerformInteractionRequest::ProcessUIResponse(
       }
     }
   }
-
-  const SmartObject* response_params = msg_params.empty() ? NULL : &msg_params;
-  if (mobile_apis::InteractionMode::BOTH != interaction_mode_ &&
-      HmiInterfaces::STATE_NOT_AVAILABLE != ui_interface_state) {
-    DisablePerformInteraction();
-    SendResponse(result,
-                 MessageHelper::HMIToMobileResult(ui_result_code_),
-                 ui_info_.empty() ? NULL : ui_info_.c_str(),
-                 response_params);
-    return true;
-  }
   return false;
```

Comment:
@AGritsevich Please remove return value because after your changes it isn't necessary.